config/config.go

```diff
@@ -5,17 +5,17 @@ import "fmt"
 type Config struct {
 	BaseConfig `mapstructure:",squash"`
 
-	HTTP       HTTPConfig     `mapstructure:"http"`
-	Panacea    PanaceaConfig  `mapstructure:"panacea"`
-	AWSS3      AWSS3Config    `mapstructure:"aws-s3"`
-	Enclave    EnclaveConfig  `mapstructure:"enclave"`
-	EndPoint   EndPointConfig `mapstructure:"endpoint"`
+	HTTP    HTTPConfig    `mapstructure:"http"`
+	Panacea PanaceaConfig `mapstructure:"panacea"`
+	AWSS3   AWSS3Config   `mapstructure:"aws-s3"`
+	Enclave EnclaveConfig `mapstructure:"enclave"`
 }
 
 type BaseConfig struct {
 	LogLevel              string `mapstructure:"log-level"`
 	ValidatorMnemonic     string `mapstructure:"validator-mnemonic"`
 	DataEncryptionKeyFile string `mapstructure:"data-encryption-key-file"`
+	EndPoint              string `mapstructure:"endpoint"`
```

Comment:
I think it would be good to put this value into the `HTTPConfig` as Tendermint does. Also, please rename it to `Endpoint`.

Reply:
Or, `ExternalEndpoint`, whatever you prefer.

Reply:
I just saw comments in the other PR: https://github.com/medibloc/panacea-data-market-validator/pull/65#discussion_r840339468.
I think it's better to minimize the `BaseConfig`. The `Endpoint` may be tightly related to HTTP settings.

Reply:
Ok I'll change it. URL is actually start with http, so I agree with your comments.